.github/workflows/example-project.yml

```diff
@@ -0,0 +1,83 @@
+name: Build example project
+
+on: [push, pull_request]
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+
+  example-project:
+
+    strategy:
+      matrix:
+        include:
+          - os: macos-latest
+          - os: windows-latest
+            toolchain-suffix: -gnu
+          - os: windows-latest
+            toolchain-suffix: -msvc
```

Comment:
We are not considering `Windows` anymore, so we can remove these lines.

Reply:
> I thought we were only postponing compiling and running a C program on Windows.

Yep, so let's add those configurations in another PR, in that way we only have `Linux` and `macOS` stuff in this one. 

> But I think there is already some value in running `cargo cbuild` on Windows (both GNU and MSVC), isn't it?
> 
> In fact, the whole reason why I started working on this topic (#113) is a build failure on MSVC, which has been fixed by specifying the MSVC binaries path, which was a really helpful information for my own project, and I think it would be helpful for others as well.

We are not dealing with `Windows` stuff in this PR, so I propose to add the `cargo cbuild` check in another PR

Reply:
For me, Windows is the main reason why I even created this PR.

What is the advantage of removing the Windows stuff from this PR?

To me, this just seems like unnecessary work.

Reply:
Since most of the `Windows` things won't be dealt dealt in this PR (due to the previous messages above), I'd rather handle **all** `Windows` stuff in a completely different PR, even the `cargo cbuild` check. I'm not saying to discard all your work, just move and extend it in another PR.

In that way, it will be much easier to address problems since they refer to different concerns.

Reply:
> move and extend it in another PR.

I will not be able to extend it, so it won't be possible to do "**all** `Windows` stuff in a completely different PR" anyway (unless my contribution is discarded).
So there is no reason to split this PR into two parts.

I think the Windows stuff that I've added here is useful on its own, that fact that it is not "complete" doesn't matter to me.

And if I *would* split this PR into two, the second one would be "incomplete", and this whole discussion would start again ...

> In that way, it will be much easier to address problems since they refer to different concerns.

I don't believe that's true.

If you see problems with this PR, I'm happy to try and fix them, it not, let's merge!

Reply:
Someone else will iterate over this in another PR, fine!